### ISSUE TITLE
"All submissions" page is missing detail links.

### ISSUE THREAD

**Issue by chrisvanrun** [comic/grand-challenge.org#3690]
**Is your feature request related to a problem? Please describe.**
An alternative approach to the submissions & evaluations is the All Submissions. However, this seems to be missing some links to the submissions and evaluation details.

**Describe the solution you'd like**
Add the evaluation & submission links to it, or add a link to the Submissions & Evaluations admin page.

**Describe alternatives you've considered**
All submissions are a collection of submissions for all phases. As such, it's nice to have them scoped per phase in the challenge admin.

**Additional context**
In the workshop I saw several users ending up on this page, with a visible error message, but no obvious way to get to the evaluation logs itself.

**Comment by jmsmkn** [comment]
Should those items be in the list anyway? I thought it would have been controlled with the `view_submission` permission?

**Comment by amickan** [comment]
Yes, the table will only contain submissions that the participant has view permission for, but not all of those will have a visible evaluation for the participant.

**Comment by jmsmkn** [comment]
But then the evaluations are not displayed in that case either, right?
